cmd/pocket_core/main.go

```diff
@@ -20,6 +23,8 @@ func main() {
 "startClient" Starts the client with the given initial configuration.
  */
 func startClient(){
+	logs.LogConstructorAndLog("test.log","pocket-core/cmd/pocket_core/main.go",
```

Comment:
Is there a way to get the file or class name dynamically? Having to type that every time seems too inconvenient.